pkg/kubectl/cmd/get.go

```diff
@@ -201,6 +203,15 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 
 	// handle watch separately since we cannot watch multiple resource types
 	isWatch, isWatchOnly := cmdutil.GetFlagBool(cmd, "watch"), cmdutil.GetFlagBool(cmd, "watch-only")
+
+	var includeUninitialized bool
+	if (isWatch || isWatchOnly) && len(args) == 2 {
+		// include the uninitialized objects by default
+		// unless explicitly set --include-uninitialized=false
+		includeUninitialized = true
```

Comment:
`-w` should only include uninitialized objects if `--include-uninitialized=true`. See my previous [comment](https://github.com/kubernetes/kubernetes/pull/50497/files#r133646291).

Reply:
Also please update the flag comment for `-w`.